exercises/matrix/matrix_test.exs

```diff
@@ -0,0 +1,66 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("matrix.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure trace: true, exclude: :pending
+
+defmodule MatrixTest do
+  use ExUnit.Case
+
+  @input "1 2 3\n4 5 6\n7 8 9"
+
+  #@tag :pending
+  test "reading from and writing to string" do
+    matrix = MatrixStruct.from_string(@input)
+
+    assert matrix != nil
```

Comment:
Well, I actually like the reading from and writing to a string, but I think we can have just a really minimal setup for that test. We're already giving some hints that we'd like them to use a struct to represent this data in the typespecs, so I don't have a problem being really broad here. Something like this should be sufficient

```elixir
matrix = Matrix.from_string(@input)
assert Matrix.to_string(matrix) == @input
```

I like the other tests as well which sort of guide the implementation without being too specific. It should end up being a really good exercise, I think!